tests/test_admin.py

```diff
@@ -445,6 +445,171 @@ def run_in_new_loop(coro):
         loop.close()
 
 
+def test_approve_platform_membership_updates_db(
```

Comment:
These tests need to be updated to reflect how admins need specific roles to act on platforms/groups - see my updated tests of `get_users` for examples.